streamlabs4j/src/main/java/com/github/twitch4j/streamlabs4j/api/domain/StreamlabsUser.java

```diff
@@ -65,6 +65,11 @@
          */
         private String displayName;
 
+        /**
+         * Primary platform selected by the user
+         */
+        private String primary;
```

Comment:
Not seeing this in the [docs](https://dev.streamlabs.com/reference#user) even though it's sent in the response.

Could we amend the javadoc to state that this field is unofficial?